sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -2229,6 +2229,12 @@
         } else if (entry.getContent() == null) {
             logger.info("entry.getContent() is null. The entity may not exist. {}", entry);
             return new SimpleResponse<>(response.getRequest(), response.getStatusCode(), response.getHeaders(), null);
+        } else if (entry.getContent().getQueueDescription() == null) {
+            final TopicDescriptionEntry entryTopic = deserialize(response.getValue(), TopicDescriptionEntry.class);
+            if (entryTopic != null && entryTopic.getContent() != null && entryTopic.getContent().getTopicDescription() != null) {
+                throw logger.logExceptionAsError(
+                    new RuntimeException(String.format("'[%s]' is not a queue, it is a topic.", entryTopic.getTitle())));
```

Comment:
Rather than a generic RuntimeException, can we throw HttpResponseException? The javadocs `@throws` needs to reflect this change.

Reply:
We cannot throw HttpResponseException because we don't have a HTTPResponse.
I think that we can handle this case as done in the [previous if block](https://github.com/Azure/azure-sdk-for-java/pull/19513/files/457d911af6e128fb7d963151f99d2d748e79e167#diff-8b1de19ce3e7548d62bea61e56b8d6dd9e11eeb8951f3afc1b5b09b9cf798e5cL2309-L2310).

Then the code can be changed from:
```java
throw logger.logExceptionAsError(new RuntimeException(String.format("'[%s]' is not a queue, it is a topic.", entryTopic.getTitle())));
```
to:
```java
logger.info("'{}' is not a queue, it is a topic.", entryTopic.getTitle());
return new SimpleResponse<>(response.getRequest(), response.getStatusCode(), response.getHeaders(), null);
```

If you agree, I will apply this change.

Reply:
@conniey do you agree?

Reply:
That is inline with what the existing logic does, so yes.